deploy/cromwell/docker-compose/README.md

```diff
@@ -0,0 +1,9 @@
+# Deploy Job Manager against Cromwell with docker-compose
+
+1. Prepare the config files following the instructions, based on your requirements.
```

Comment:
Following what instructions? Could we link to them?

Reply:
👍 Will link to the other readmes.